cmd/clef/main.go

```diff
@@ -351,6 +378,64 @@ func attestFile(ctx *cli.Context) error {
 	return nil
 }
 
+func listAccounts(c *cli.Context) error {
+	if err := initialize(c); err != nil {
+		return err
+	}
+	// listaccounts is meant for users using the CLI.
+	var (
+		ui                        = core.NewCommandlineUI()
+		pwStorage storage.Storage = &storage.NoStorage{}
+		ksLoc                     = c.String(keystoreFlag.Name)
+		lightKdf                  = c.Bool(utils.LightKDFFlag.Name)
+	)
+	am := core.StartClefAccountManager(ksLoc, true, lightKdf, "")
+	// Access external API and call List()
+	api := core.NewSignerAPI(am, 0, true, ui, nil, false, pwStorage)
+	internalApi := core.NewUIServerAPI(api)
```

Comment:
This chunk of code is now repeated at least three times. I suggest moving this to it's own function, with the signature 
```
func initInternal(c *cli.Context) (*core.UIServerAPI, error) {
```
or maybe `initInternalApi`. 
And then shorten the three methods , e.g 
```golang
func newAccount(c *cli.Context) error {
	internalApi, err := initInternal(c)
	if err != nil {
```
And then move these two new methods down, so they appear after `newAccount` within the file.